docs/source/commandline/commandline-all.rst

```diff
@@ -297,13 +533,54 @@ Example:
     includes media types (cartridge, cassette, diskette and more) as well as
     common file extensions which are supported.
 
+    Example:
+        .. code-block:: bash
+
+            mame64 coco3 -listmedia
+            SYSTEM           MEDIA NAME       (brief)    IMAGE FILE EXTENSIONS SUPPORTED
+            ---------------- --------------------------- -------------------------------
+            coco3            cassette         (cass)     .wav  .cas
+                             printout         (prin)     .prn
+                             cartridge        (cart)     .ccc  .rom
+                             floppydisk1      (flop1)    .dmk  .jvc  .dsk  .vdk  .sdf  .os9  .d77  .d88  .1dd  .dfi  .hfe  .imd  .ipf  .mfi  .mfm  .td0  .cqm  .cqi
+                             floppydisk2      (flop2)    .dmk  .jvc  .dsk  .vdk  .sdf  .os9  .d77  .d88  .1dd  .dfi  .hfe  .imd  .ipf  .mfi  .mfm  .td0  .cqm  .cqi
+                             harddisk1        (hard1)    .vhd
+                             harddisk2        (hard2)    .vhd
+
 .. _mame-commandline-listsoftware:
 
 **-listsoftware** / **-lsoft** [<*pattern*>]
 
-    Posts to screen all software lists which can be used by the entered
-    *pattern* or system. Note that this is simply a copy/paste of the .XML file
-    which reside in the HASH folder which are allowed to be used.
+    Displays the contents of all softlists that can be used by the system or
```

Comment:
It’s better to use the “software list” rather than “softlist” in user-facing documentation.

Reply:
Valid point. Corrected.